### PR TITLE
Prereq divs

### PR DESCRIPTION
**Pull request by aaren**
Addresses #25. Also blocking #22.

Previously, Div creation was triggered by a Blockquote that began with a
header that contained a single class that was in a list of special
classes.

Now, Div creation is _also_ triggered by a Blockquote that begins with a
header that has it's text contained in a list of special titles. The
title is used to lookup an appropriate class to give the Div.

In particular, 'prerequisites' is a special title, giving the class
'prereq'.

This input:

```
> ## Prerequisites
>
> A short paragraph describing what learners need to know before
> tackling this lesson.
```

will trigger this output:

```
<div id="prerequisites" class="prereq">
<h2>Prerequisites</h2>
<p>A short paragraph describing what learners need to know before
tackling this lesson.</p>
</div>
```

### CODE REVIEW

**Comment by aaren** [comment]
@r-gaia-cs can you review?

File: tools/blockquote2div.py [hunk 1]
```diff
@@ -34,10 +34,17 @@
...
 
-valid_classes = ['objectives', 'callout', 'challenge']
+# These are classes that, if set on the title of a blockquote, will
+# trigger the blockquote to be converted to a div.
+special_classes = ['objectives', 'callout', 'challenge']
 
+# These are titles of blockquotes that will cause the blockquote to
+# be converted into a div. They are 'title': 'class' pairs, where the
+# 'title' will create a blockquote with the corresponding 'class'.
+special_titles = {'prerequisites': 'prereq'}
```
Comment (aaren) [inline tools/blockquote2div.py]: we could also have 'challenge', 'callout', 'objectives' as special titles.

Comment (rgaiacs) [inline tools/blockquote2div.py]: Yes! Please.

**Comment by aaren** [comment]
@r-gaia-cs any more comments? shall I squash into a single commit?

**Comment by rgaiacs** [comment]
> any more comments?

Thanks very much for work on this. =)

> shall I squash into a single commit?

No need to do it. I'm merging it now.

**Pull request merged**